Formula/git-annex.rb

```diff
@@ -3,6 +3,7 @@ class GitAnnex < Formula
   homepage "https://git-annex.branchable.com/"
   url "https://hackage.haskell.org/package/git-annex-8.20200720.1/git-annex-8.20200720.1.tar.gz"
   sha256 "d6252697151ab01cc8f7a492dc5eb4b4e773c0d48771df360c82a7cca7a39c3c"
+  license "AGPL-3.0-or-later"
```

Comment:
> Parts of git-annex are licensed under the GPL, BSD, and other licenses. For details, see [this file](http://source.git-annex.branchable.com/?p=source.git;a=blob_plain;f=COPYRIGHT;hb=HEAD).

Do we want to leave a comment to add these once Homebrew supports them?

Reply:
yeah, that was a little bit disagreement I had earlier.

According to https://hackage.haskell.org/package/git-annex-8.20200720.1, the license probably should be `AGPL-3.0-only`

Reply:
This definitely shouldn't block this PR as the brew PR to allow multiple license requirements (and new version) hasn't been completed yet.